### PR TITLE
fix: img payload in firefox

### PR DESCRIPTION
**Pull request by zhouhanseng**
Fix the bug that there's no 'img payload' in firefox browser.

### CODE REVIEW

**Review by guanbinrui — APPROVED** [review]

File: src/extension/injected-script/addEventListener.ts [hunk 1]
```diff
@@ -53,7 +53,8 @@ import type { CustomEvents } from './CustomEvents'
                 document.activeElement!.dispatchEvent(e)
                 return undefined!
             } else if (textOrImage.type === 'image') {
-                const xray_binary = globalThis.window.Uint8Array.from(textOrImage.value)
+                const Uint8Array = globalThis.Uint8Array ? globalThis.Uint8Array : globalThis.window.Uint8Array
```
Comment (zhouhanseng) [inline src/extension/injected-script/addEventListener.ts:56]: There's a `globalThis.Uint8Array` in firefox. `globalThis.window.Uint8Array` is undifined in firefox

File: webpack.config.js [hunk 2]
```diff
@@ -111,7 +111,6 @@ module.exports = (argvEnv, argv) => {
         let resolution = 'desktop'
         if (target.Chromium) buildTarget = 'chromium'
         if (target.Firefox) buildTarget = 'firefox'
-        if (target.FirefoxDesktop) firefoxVariant = 'fennec'
```
Comment (zhouhanseng) [inline webpack.config.js:114]: I think `FirefoxDesktop` is not 'fennec', https://wiki.mozilla.org/Mobile/Fennec

File: src/extension/injected-script/addEventListener.ts (hunk 1 above)
Comment (Jack-Works) [reply src/extension/injected-script/addEventListener.ts:56]: 🤔🤔🤔🤔🤔🤔🤔🤔🤔

File: webpack.config.js (hunk 2 above)
Comment (Jack-Works) [reply webpack.config.js:114]: IGTM

**Pull request merged**